packages/components/src/drop-zone/index.tsx

```diff
@@ -10,84 +10,32 @@ import { __ } from '@wordpress/i18n';
 import { useState } from '@wordpress/element';
 import { upload, Icon } from '@wordpress/icons';
 import { getFilesFromDataTransfer } from '@wordpress/dom';
-import {
-	__experimentalUseDropZone as useDropZone,
-	useReducedMotion,
-} from '@wordpress/compose';
+import { __experimentalUseDropZone as useDropZone } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
-import {
-	__unstableMotion as motion,
-	__unstableAnimatePresence as AnimatePresence,
-} from '../animation';
 import type { DropType, DropZoneProps } from './types';
 import type { WordPressComponentProps } from '../context';
-
-const backdrop = {
-	hidden: { opacity: 0 },
-	show: {
-		opacity: 1,
-		transition: {
-			type: 'tween',
-			duration: 0.2,
-			delay: 0,
-			delayChildren: 0.1,
```

Comment:
Are we taking this into consideration? Could be done with `transition-delay` FWIW.

Reply:
Yes, we are; the additional keyframe at 50% in the scale / scaleOut keyframes is what implements this delay. It's very close enough to the original behavior. However, as you suggest, `transition-delay` + transition might be an easier and more intuitive way of implementing the same 👍🏻